modules/dbfs/mysql.go

```diff
@@ -440,7 +440,7 @@ func (di *DatabaseImpl) MySQLFileCreate(username string, filename string, relati
 	for rows.Next() {
 		err = rows.Scan(&fileID)
 		if err != nil {
-			return -1, err
+			return -1, ErrNoDbChange
```

Comment:
that's a go thing, things that start with an upper case letter are exported from the package